main/tests/server/src/com/google/refine/exporters/CsvExporterTests.java

```diff
@@ -224,6 +224,66 @@ public void exportCsvWithEmptyCells() {
                 ",row2cell1,row2cell2\n");
     }
 
+    /*
+    check if CSV exporter can handle a semi-colon as a custom separator
+     */
+    @Test
+    public void exportCsvWithCustomSeparatorSemicolon() {
```

Comment:
I'd be surprised if this adds any coverage of our code, but maybe I'm wrong! What is there that's specific to semicolons which make this worth testing?